### PR TITLE
Fix lsvmbus against ORACLE, Debian, CoreOS and ClearOS.

### PR DESCRIPTION
**Pull request by LiliDeng**
Use python lsvmbus version for those distro which no hyperv-tool package.
https://raw.githubusercontent.com/torvalds/linux/master/tools/hv/lsvmbus
Below is result from my local run, @Tong-Xu Please help double confirm the results.

image

### CODE REVIEW

File: Testscripts/Linux/LSVMBUS.sh [hunk 1]
```diff
@@ -37,13 +37,30 @@ if [[ "$DISTRO" =~ "redhat" ]] || [[ "$DISTRO" =~ "centos" ]]; then
     if ! rpm -q hyperv-tools; then
         yum install -y hyperv-tools
     fi
+    distro=$(grep -ihs "ORACLE" /etc/*release)
+fi
+
+if [[ "$DISTRO" =~ "debian" ]] || [[ "$DISTRO" =~ "clear-linux-os" ]] || [[ "$distro" != "" ]]; then
+    install_package wget
```
Comment (lizzha) [inline Testscripts/Linux/LSVMBUS.sh]: [[ "$distro" != "" ]] already includes [[ "$DISTRO" =~ "debian" ]] || [[ "$DISTRO" =~ "clear-linux-os" ]]

Comment (lizzha) [reply Testscripts/Linux/LSVMBUS.sh]: why not checking for the existence of command lsvmbus? And only download if it doesn't exist

Comment (LiliDeng) [reply Testscripts/Linux/LSVMBUS.sh]: $distro is not $DISTRO, I added it to distingue Oracle and Redhat. Ok, I will update logic to check if the lsvmbus not exist by default.

**Comment by Tong-Xu** [comment]
Use python lsvmbus version for those distro which no hyperv-tool package.
The following is the results:
[LISAv2 Test Results Summary]
Test Run On           : 01/24/2019 08:42:54
ARM Image Under Test  : credativ : Debian : 9-backports : Latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:12

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 LSVMBUS                                                                           PASS                 3.83 
[LISAv2 Test Results Summary]
Test Run On           : 01/24/2019 09:42:43
ARM Image Under Test  : Oracle : Oracle-Linux : 7.6 : Latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:9

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 LSVMBUS                                                                           PASS                 6.67 
[LISAv2 Test Results Summary]
Test Run On           : 01/24/2019 08:43:37
ARM Image Under Test  : CoreOS : CoreOS : Stable : Latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:11

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 LSVMBUS                                                                           PASS                 3.27 
[LISAv2 Test Results Summary]
Test Run On           : 01/24/2019 08:43:58
ARM Image Under Test  : clear-linux-project : clear-linux-os : basic : latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:13

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 LSVMBUS                                                                           PASS                 4.18 
[LISAv2 Test Results Summary]
Test Run On           : 01/24/2019 08:53:08
ARM Image Under Test  : OpenLogic : CentOS : 7.5 : Latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:14

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 LSVMBUS                                                                           PASS                 5.36 
[LISAv2 Test Results Summary]
Test Run On           : 01/24/2019 08:56:33
ARM Image Under Test  : RedHat : RHEL : 7.5 : Latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:13

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 LSVMBUS                                                                           PASS                  4.7 
[LISAv2 Test Results Summary]
Test Run On           : 01/24/2019 09:08:55
ARM Image Under Test  : SUSE : SLES-Standard : 15 : Latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:17

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 LSVMBUS                                                                           PASS                 3.76 
[LISAv2 Test Results Summary]
Test Run On           : 01/24/2019 10:04:49
ARM Image Under Test  : canonical : ubuntuserver : 18.04-lts : Latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:12

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 LSVMBUS                                                                           PASS                 3.66

**Comment by LiliDeng** [comment]
Please review again @lizzha and already tested against latest code.
Ubuntu 18.04, Debian 9 back-port, Oracle 76, CoreOS - Pass

**Pull request merged**